app/konnect/updates.md

```diff
@@ -9,6 +9,19 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
+## November 2023
+
+**Multi-geo support**
```

Comment:
❗ For review: I'm not sure if this is accurate based on how the release will be rolled out. Is this feature going to be released in the following order?

1. AU geo available in beta
2. Multi-geo available to new customers
3. multi-geo available to all customers

Reply:
We are not release AU geo in beta. Instead, we are releasing AU geo with:
1. Multi-geo available to all new signups, including the AU geo
2. Multi-geo available to existing Konnect customers, including the AU geo.